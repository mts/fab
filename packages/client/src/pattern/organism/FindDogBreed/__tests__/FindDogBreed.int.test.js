import { defaultRender } from './FindDogBreed.int.render'

jest.mock('../../../../store/thunk/completeState', () => ({ setAppCompleteStateThunk: () => {} }))

describe('<FindDogBreed />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
