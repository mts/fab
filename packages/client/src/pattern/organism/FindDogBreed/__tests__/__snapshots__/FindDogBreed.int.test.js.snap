// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FindDogBreed /> Snaphot must match defaultRender 1`] = `
<div
  className="breed"
>
  <h2>
    A random image of 
     breed
  </h2>
  <h4>
    Type in new breed, press enter or click search
  </h4>
  <div
    className="breedSearch"
  >
    <input
      className="input"
      onKeyUp={[Function]}
    />
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Search Dog
    </button>
  </div>
  <div
    className="breedShow"
  >
    <div
      className="image"
    >
      <img
        alt="random"
        src="https://via.placeholder.com/300"
      />
    </div>
  </div>
</div>
`;
